### PR TITLE
LORE-187: Update Transformation API Impl to use REST instead of queues.

### PR DESCRIPTION
**Pull request by figliold**
#### What does this PR do?
Updates Transformation API Impl to use REST instead of queues.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
 @cjlange
 @mdang8
 @josephthweatt
 @brendan-hofmann
 @mcalcote

#### What are the relevant tickets?
Fixes #LORE-187

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by brendan-hofmann** [comment]
image nice!

File: distros/spring/src/main/resources/application-dev.yml [hunk 1]
```diff
@@ -1,12 +1,6 @@
-spring:
-  rabbitmq:
-    user: ion
-    password: ion
-    host: localhost
-    port: 5672
-    virtualHost: /
-    exchange: transformation
-    routingKey: request
+endpoints:
```
Comment (cjlange) [inline distros/spring/src/main/resources/application-dev.yml]: @figliold - ❗Can you add the copyright header to this file, please?

File: distros/spring/src/main/resources/application-int.yml [hunk 2]
```diff
@@ -1,12 +1,6 @@
-spring:
-  rabbitmq:
-    user: ion
-    password: ion
-    host: rabbitmq
-    port: 5672
-    virtualHost: /
-    exchange: transformation
-    routingKey: request
+endpoints:
```
Comment (cjlange) [inline distros/spring/src/main/resources/application-int.yml]: @figliold - ❗Can you add the copyright header to this file, please?

**Review by cjlange — CHANGES_REQUESTED** [review]

**Review by mcalcote — APPROVED** [review]

**Review by mdang8 — APPROVED** [review]

**Review by brendan-hofmann — APPROVED** [review]

File: distros/spring/src/main/resources/application-dev.yml (hunk 1 above)
Comment (figliold) [reply distros/spring/src/main/resources/application-dev.yml]: added

File: distros/spring/src/main/resources/application-int.yml (hunk 2 above)
Comment (figliold) [reply distros/spring/src/main/resources/application-int.yml]: added

**Review by cjlange — APPROVED** [review]

**Pull request merged**